MVCapplication/src/main/webapp/WEB-INF/views/product.jsp

```diff
@@ -15,433 +18,36 @@
-                        <td class="center">-</td>
-                        <td class="center">C</td>
-                    </tr>
-                    <tr class="gradeA">
-                        <td>Misc</td>
-                        <td>NetFront 3.4</td>
-                        <td>Embedded devices</td>
-                        <td class="center">-</td>
-                        <td class="center">A</td>
-                    </tr>
-                    <tr class="gradeX">
-                        <td>Misc</td>
-                        <td>Dillo 0.8</td>
-                        <td>Embedded devices</td>
-                        <td class="center">-</td>
-                        <td class="center">X</td>
-                    </tr>
-                    <tr class="gradeX">
-                        <td>Misc</td>
-                        <td>Links</td>
-                        <td>Text only</td>
-                        <td class="center">-</td>
-                        <td class="center">X</td>
-                    </tr>
-                    <tr class="gradeX">
-                        <td>Misc</td>
-                        <td>Lynx</td>
-                        <td>Text only</td>
-                        <td class="center">-</td>
-                        <td class="center">X</td>
-                    </tr>
-                    <tr class="gradeC">
-                        <td>Misc</td>
-                        <td>IE Mobile</td>
-                        <td>Windows Mobile 6</td>
-                        <td class="center">-</td>
-                        <td class="center">C</td>
-                    </tr>
-                    <tr class="gradeC">
-                        <td>Misc</td>
-                        <td>PSP browser</td>
-                        <td>PSP</td>
-                        <td class="center">-</td>
-                        <td class="center">C</td>
-                    </tr>
-                    <tr class="gradeU">
-                        <td>Other browsers</td>
-                        <td>All others</td>
-                        <td>-</td>
-                        <td class="center">-</td>
-                        <td class="center">U</td>
-                    </tr>--%>
-                    </tbody>
-                </table>
+                        </thead>
+                        <tbody>
+                        <c:forEach items="${products}" var="product">
+                            <tr class="gradeA">
+                                <td><c:out value="${product.name}"></c:out></td>
+                                <td><c:out value="${product.description}"></c:out></td>
+                                <td><c:out value="${product.category.name}"></c:out></td>
+                                <td><c:out value="${product.unit.name}"></c:out></td>
+                                <td><c:out value="${product.image}"></c:out></td>
+                                <td><c:out value="${product.image}"></c:out></td>
```

Comment:
instead of 
`<c:out value="${product.image}"></c:out>`
can be just
`${product.image}`